tests/test-location.py

```diff
@@ -22,6 +22,7 @@
 import subprocess
 import tempfile
 import unittest
+import uuid
```

Comment:
If you're using `uuid` anyway, can you convert the event ID to a UUID object directly, and use whatever comparison facility the UUID library gives you?